src/Playwright.NUnit/PlaywrightTest.cs

```diff
@@ -22,15 +22,14 @@
  * SOFTWARE.
  */
 
-using System;
 using System.Threading.Tasks;
 using NUnit.Framework;
 
 namespace Microsoft.Playwright.NUnit
 {
     public class PlaywrightTest : WorkerAwareTest
     {
-        public static string BrowserName => (Environment.GetEnvironmentVariable("BROWSER") ?? Microsoft.Playwright.BrowserType.Chromium).ToLower();
+        public string BrowserName { get; internal set; } = null!;
```

Comment:
This is a breaking change, isn't it? I believe we should support both, probably giving priority to the env variable.

Reply:
Ok, I see that it moved to the settings parser.